src/engine/Drawing/SpriteSheet.ts

```diff
@@ -186,6 +213,25 @@ export class SpriteSheetImpl {
     anim.drawHeight = maxHeight;
     return anim;
   }
+
+  /**
+   * Builds sprites array with dimensions and spacing for loaded image
+   */
+  private _buildSpritesArray() {
+    const spacing = this.getSpacingDimensions();
+
+    for (let row = 0; row < this.rows; row++) {
+      for (let col = 0; col < this.columns; col++) {
+        this.sprites[col + row * this.columns] = new Sprite(
```

Comment:
@eonarheim Any implications to leaving old Sprites around from the array?

Reply:
I think this is fine, legacy sprites do have some canvas overhead internally but this shouldn't be on a hot path like rebuilding every frame or something.

Reply:
In the future, zero potential problems at all, sprites will be a "view" into a source image

Reply:
What about references to these sprites from the other public methods? Should it actually set the properties of the sprite instead of overwriting with new instance?

Reply:
@kamranayub Perhaps, although I'm not sure I see a use case for modifying sprites underneath users. You mentioned on another thread, but SpriteSheet's should probably be immutable after creation